client/js/components/fragments/Map.js

```diff
@@ -0,0 +1,80 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router';
+import GoogleMapsLoader from 'google-maps';
+
+class Map extends Component {
+  constructor () {
+    super();
+    this.map;
```

Comment:
please check, as as far as I know (I'm no JS expert) such construct does not do anything